token/cli/src/config.rs

```diff
@@ -355,8 +362,11 @@ impl<'a> Config<'a> {
         address_name: &str,
         wallet_manager: &mut Option<Arc<RemoteWalletManager>>,
     ) -> Result<Pubkey, Error> {
-        if let Some(address) = pubkey_of_signer(arg_matches, address_name, wallet_manager).unwrap()
-        {
+        let address = match pubkey_of_signer(arg_matches, address_name, wallet_manager) {
+            Ok(address) => address,
+            Err(e) => return Err(e.to_string().into()),
+        };
```

Comment:
nit: how about bubbling this in one line by mapping the error?
```suggestion
        let address = pubkey_of_signer(arg_matches, address_name, wallet_manager).map_err(|e| e.to_string().into())?;
```

Reply:
good suggestion, updated all instances with `map_err`.